test-integration/helpers/window-position.coffee

```diff
@@ -0,0 +1,12 @@
+windowPosition = (test) ->
```

Comment:
My initial suggestion would be closures because it makes it clear there is no place to add additional state (the majority of helpers). If we wanted to store more state we could easily convert them to a class later.